### ISSUE TITLE
Activity from other apps

### ISSUE THREAD

**Issue by louis030195** [muesli/readme-scribe#4]
Hello, awesome action !

I'm wondering if it would be possible to implement something for linking automatically my latest Medium posts, or other things probably query-able by API (or scrapping) ?

If you guide me I could implement it.

Thanks !

**Comment by muesli** [comment]
Absolutely! This should be done in `markscribe`, which is the actual software that this GitHub action wraps. I'm moving this ticket to the right repository!
